devicon.json

```diff
@@ -8644,6 +8644,28 @@
         "color": "#000000",
         "aliases": []
     },
+    {
+        "name": "titaniumsdk",
+        "altnames": [],
+        "tags": [
+            "app",
+            "mobile",
+            "javascript",
+            "cross-platform",
+            "mobile",
```

Comment:
mobile is written twice
```suggestion
```

Reply:
very nice :+1: Thanks for the "suggestion" part :+1: